src/Appwrite/Event/Mail.php

```diff
@@ -133,6 +134,28 @@ public function getName(): string
         return $this->name;
     }
 
+    /**
+     * Set SMTP
+     *
+     * @param array $smtp
+     * @return self
+     */
+    public function setSmtp(array $smtp): self
```

Comment:
Same suggestions about the complex structure.

Reply:
updated